migration/migrator/migrations/system/20220710211447_switch_opencv_headless.py

```diff
@@ -0,0 +1,22 @@
+"""Migration for the Submitty system."""
+import pkg_resources
+import os
+
+def up(config):
+    #get existing installed packages and check if opencv-python is there
+    #https://stackoverflow.com/questions/44210656/how-to-check-if-a-module-is-installed-in-python-and-if-not-install-it-within-t    
+    installed = {pkg.key for pkg in pkg_resources.working_set}
+    if 'opencv-python' in installed:
+        print("Uninstalling opencv-python")
+        os.system("pip3 uninstall opencv-python --yes --no-input")
+
+    os.system("pip3 install pip install opencv-python-headless==4.6.0.66")
```

Comment:
So a couple things:

1. Can we use `subprocess.check_call` instead of `os.system` so that if these lines fail to do what we want, we're not potentially left in an "invalid" state where we still use `opencv-python` instead of `opencv-python-headless`.
1. I'd suggest using `python3 -m pip ...` to invoke pip as opposed to using a bare `pip3 ...`. It's the recommended way to use it according to [pip's docs](https://pip.pypa.io/en/stable/getting-started/#install-a-package-from-github). It's mainly to accomdiate systems where you may have multiple python installations, and avoid calling the wrong pip, but I think it's a good habit to have nevertheless